### ISSUE TITLE
"vue/no-unused-vars" cannot set a value

### ISSUE THREAD

**Issue by hongl-1** [vuejs/eslint-plugin-vue#2010]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:**
- **eslint-plugin-vue version:**
- **Node version:**
- **Operating System:**

**Please show your full configuration:**

```json5
{
//  other code
  rules: {
    'vue/no-unused-vars': ['error', {
      ignorePattern: '^_'
    }]
  },
//  other code
}
```

**What did you do?**

```vue
when I run: yarn eslint src/App,vue
get error info: 
**Configuration for rule "vue/no-unused-vars" is invalid:                                                                          
 Value [{"ignorePattern":"^_"}] should NOT have more than 0 items.**  
```

**What did you expect to happen?**

**What actually happened?**

**Repository to reproduce this issue**
